core/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js

```diff
@@ -38,7 +38,8 @@ oppia.directive('collectionSummaryTile', [
         isPlaylistTile: '&isPlaylistTile',
         showLearnerDashboardIconsIfPossible: (
           '&showLearnerDashboardIconsIfPossible'),
-        isContainerNarrow: '&containerIsNarrow'
+        isContainerNarrow: '&containerIsNarrow',
+        shouldAddToPlaylistIcon: '&shouldAddToPlaylistIcon',
```

Comment:
I remember commenting on the naming of this in the previous review, but it doesn't look like it's been addressed even though the reply said "understood". In particular I can't parse "should add to playlist icon" in a way that makes sense.

Please see my comments on #4764 and make sure that all of them are addressed. Once you've done that I'll be happy to take another look at this PR.

Reply:
Oh! My bad, I've updated the variable as ```belongsToCreatorPlaylist``` and the code accordingly. It now checks if the activity belongs to creator's playlist, then it should not show the playlist icon. I believe it makes more sense now.